resources/test/valid/chainspec.toml

```diff
@@ -220,7 +220,7 @@ max_ttl = '35years'
 max_dependencies = 36
 max_block_size = 37
 block_max_deploy_count = 375
-block_max_transfer_count = 375
+block_max_transfer_count = 376
```

Comment:
This seems to be missing from the chainspec `check_bundled_spec` test then.
(We should somehow refactor that test so that we can't forget adding new chainspec fields…)